.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "fin"]
     path = fin
     url = https://github.com/ROCmSoftwarePlatform/MIFin.git
-    branch = develop
+    branch = cderb/param_check
```

Comment:
@cderb Why do you need this branch?

Reply:
Guess it might need some cleaning up?

Reply:
There is a chicken egg problem.  The 'TestPerfCfgParams' function was renamed so the Fin PR that changes the function name can't pass, because the MIOpen function doesn't exist.
At the same time, this PR which renames the function can't pass, because Fin has the old function name.
This branch has the new function. It will be returned to develop shortly after this PR settles.

Reply:
Does this mean that every time the API changes, we have to do this trick? If yes, then you need to do something with mutual dependencies.

It might be worth adding the new function to the `MIOpen` first, while keeping the old one, then enabling it in the `Fin`, and then deleting the old function in the `MIOpen`.

Reply:
It would happen anytime a function was removed from the API, yes.

I'd prefer to remove this function in this PR as opposed leaving bloat. Alternatively I can also disable the fin build phase until this is resolved.

Reply:
@cderb I'm asking because I'm going to do the same thing with `AnySolver`. Its API will be changed, but I plan to keep methods with old signatures before they are implemented in the `Fin`.

Reply:
Resolved this on the side of Fin.